AraEvent/AraEventCalibrator.cxx

```diff
@@ -1833,11 +1833,14 @@ void AraEventCalibrator::loadAtriPedestals(AraStationId_t stationId)
     // now, we open and load the pedestal files
     gzFile inPed = gzopen(fAtriPedFile[calibIndex], "r");
     std::vector<char> buffer(6000000);
-    int nRead = gzread(inPed, &buffer[0], sizeof(char)*6000000);
+    int nRead = gzread(inPed, buffer.data(), buffer.size());
```

Comment:
And the prior was bad because it's just some giant "glob" which can grab data that doesn't exist?

Reply:
Okay, got it.